sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -235,6 +242,8 @@ clients_summary AS (
     environment.build.build_id AS env_build_id,
     environment.build.version AS env_build_version,
     environment.build.architecture AS env_build_arch,
+    environment.build.platform_version AS env_build_platform_version,
```

Comment:
Same case of xpcom_abi, just a double check that data from environment.build (main ping) is consistent with data from application (new_profile and first_shutdown pings)